src/test/progpow_tests.cpp

```diff
@@ -0,0 +1,166 @@
+#include <test/test_veil.h>
```

Comment:
Change all `auto`s in file.

Reply:
I don't think messing with the autos in these tests is going to help much. If anything, I would think we would like to keep the progpow tests as is, so we can easily pull in upgrades and fixes without any issues.